### PR TITLE
allow reconnect option send to node-ldap client

### PR DESCRIPTION
**Pull request by cgalibern**
Call hanging after ldap server close connexion #23

### CODE REVIEW

File: lib/ldapauth.js [hunk 1]
```diff
@@ -132,6 +134,10 @@ function LdapAuth(opts) {
     maxIdleTime: opts.maxIdleTime
   };
 
+  if (opts.reconnect) {
```
Comment (vesse) [inline lib/ldapauth.js]: No need to check, just add to `clientOpts` like the other options are added.

**Comment by vesse** [comment]
Sorry for the delay, I should not read emails if unable to check the pull request straight away... But, just add the option to other `clientOpts` and I'll merge & publish.

**Comment by cgalibern** [comment]
ok thanks

**Comment by cgalibern** [comment]
@vesse thanks

**Pull request merged**